docs/AppFramework.md

```diff
@@ -223,7 +223,7 @@ By default, the App Framework polls the remote object storage location for new o
 For more information, see the [Description of App Framework Specification fields](#description-of-app-framework-specification-fields).
 
 ## Description of App Framework Specification fields
-The App Framework configuration is supported on the following Custom Resources: Standalone, ClusterMaster, SearchHeadCluster, MonitoringConsole and LicenseManager. Configuring the App framework requires:
+The App Framework configuration is supported on the following Custom Resources: Standalone, ClusterManager, SearchHeadCluster, MonitoringConsole and LicenseManager. Configuring the App framework requires:
```

Comment:
When supporting both Manager and master should we add both in docs?

Reply:
We want new deployments to use the Manager as much as possible moving forward. The docs as well as the examples are all switched to Manager so anyone reading for the first time will start with the Manager.